howto/setup-state-store/setup-azure-cosmosdb.md

```diff
@@ -4,7 +4,7 @@
 
 [Follow the instructions](https://docs.microsoft.com/en-us/azure/cosmos-db/how-to-manage-database-account) from the Azure documentation on how to create an Azure CosmosDB account.  The database and collection must be created in CosmosDB before Dapr consumes it.  
 
-**Note : The partition key for the collection must be "/id".**
+**Note : The partition key for the collection must be "/partitionKey".  This is case-sensitive.**
```

Comment:
The way this is phrased right now it sounds like the actual value of the partition key "must be" `/partitionKey` which is not the case, right? Rewrite to make it clear that the case of the partitioning key must match the case of the document's property.

Reply:
It does have to be exactly `/partitionKey`.

Reply:
To elaborate, in general the field doesn't have to have this name, "id" and the correspoding "/id" are fine.  But this is a ripple from the requirement to support multi-item transactions.  In that context all items in the transaction must be in the same partition.  We were earlier using id for this.  But id is the key from the users view, so it must be unique.  So "partitionKey" was added as another field to represent the partition.  It's used in actor code to make sure items for the same actor end up in the same partition.